AbstractEntityValidation.md

```diff
@@ -0,0 +1,75 @@
+Feature: Validation of an abstract entity
+
+Scenario: Validation of an abstract entity
+    When sending a POST request to "/ed-fi/schools" with body
```

Comment:
`Schools` is not an abstract entity, but `educationOrganization` _is_. Suggest changing to `educationOrganization`.

Reply:
Entity changed to the expected one